packages/ListBoxBrowser/README.md

```diff
@@ -0,0 +1,173 @@
+# ListBoxBrowser
+
+It is a React component that allows users to select one or multiple options using two columns, similar to the experience of browsing a file manager in any giving Operative System.
```

Comment:
Don't think we need to start off the sentence with "It is a React component".

Reply:
feel free to modify it :)